Apps/Admin/Tests/Functional/cypress/utilities/supportUtilities.js

```diff
@@ -1,5 +1,11 @@
 import { getTableRows } from "./sharedUtilities";
 
+function selectTab(tabText) {
+    cy.get("[data-testid=patient-details-tabs]")
+        .contains(".mud-tab", tabText)
+        .click();
+}
```

Comment:
If this function was exported, it wouldn't need to be duplicated in the test files.

Reply:
Cool I'll clean up for the admin tests.